wordpress-mobile/WordPress-iOS
# Jetpack Focus: Add switch card to the menu in phase 4

Closes #19813
Design ref: qk5s4q4mZsXknwhMWo2vIY-fi-439%3A5242

## Description
This PR adds a "Switch to Jetpack" button to the bottom of the menu in phase 4. The card from phase 3 is reused here with a few modifications to match the design of the phase 4 card.

### Screenshots

| Light Mode | Dark Mode |
| - | - |
|![light](https://user-images.githubusercontent.com/25306722/210468597-e736a2f4-80df-4021-858c-6a0851cfedcf.png)|![dark](https://user-images.githubusercontent.com/25306722/210468606-328cfa05-d652-485a-a23f-dba2ea060240.png)|

## Testing Instructions

1. Go to `JetpackBrandingMenuCardPresenter.swift` line 173
2. Change the value of `secondsInDay` to any small value, 10, for example
3. Launch the app
4. Open the debug menu and enable "Jetpack Features Removal Phase Three"
5. Kill and relaunch the app
6. Make sure the menu card is displayed as the first menu item
7. Make sure the card matches the design
8. Make sure tapping the card displays the overlay
9. Tap the ellipsis and tap on "Hide this"
10. The card should be removed
11. Open the debug menu and enable "Jetpack Features Removal Phase Four" 
12. Kill and relaunch the app
13. Make sure the menu card is displayed as the last menu item
14. Make sure the card is compact and matches the design
15. Make sure tapping the card displays the overlay
16. Tap the ellipsis and tap on "Remind me later"
17. The card should be removed
18. Wait for the duration to pass (If you set `secondsInDay` to 10, then wait for 40 seconds)
19. Pull to refresh the menu
20. Make sure the menu card is displayed

## Regression Notes
1. Potential unintended areas of impact
Phase 3 card

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and `JetpackBrandingMenuCardPresenterTests`

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
